packages/sanity/playwright-ct/tests/formBuilder/inputs/PortableText/copyPaste/input.ts

```diff
@@ -5,3 +5,37 @@ export const REMOVED_INPUT_SNAPSHOT = [{"_key":"3969e2c82eb1","children":[{"_typ
​‌​﻿​‍​﻿‍‌​﻿‌‌​﻿​‍​﻿​‌‌‍​‍​‍‌‌​﻿​‍​﻿​‍​‍‌‌​﻿‌‌‌​‌​​‍﻿‍‌﻿‌​‌‍‌‌‌﻿‍​‌﻿‌​​﻿﻿﻿‌‍​‍‌‍​‌‌﻿​﻿‌‍‌‌‌‌‌‌‌﻿​‍‌‍﻿​​﻿﻿‌​‍‌‌​﻿​‍‌​‌‍‌﻿​​‌﻿​‍‌‍‌‌‌﻿​﻿‌‍‌‌‌‍﻿‍‌﻿‌​‌‍​‌‌﻿‌​‌‍‍‌‌‍﻿﻿‌‍﻿‍​‍‌‍‌‍‍‌‌‍‌​​﻿﻿‌​﻿‌﻿​﻿​‌​﻿‌‍‌‍‌‌​﻿​‍‌‍​﻿​﻿‍​‌‍​‌​‍﻿‌‌‍​‌​﻿​‌​﻿‌‍​﻿‍​​‍﻿‌​﻿‌​​﻿‍​​﻿‌‌​﻿​‍​‍﻿‌​﻿‍​​﻿‌﻿‌‍‌‌​﻿​‍​‍﻿‌​﻿​​‌‍​﻿​﻿​‍‌‍​‍​﻿‌​‌‍‌​​﻿‌​​﻿​‍‌‍​﻿​﻿‍‌​﻿‌​‌‍​‍​‍‌‍‌﻿‌​‌﻿‍‌‌﻿​​‌‍‌‌​﻿﻿‌‌﻿​﻿‌‍‍‌‌‍﻿‌‌﻿​​‌‍﻿​‌‍‌‌‌​​‍‌‍﻿​‌‍﻿﻿‌‍​﻿‌‍‍﻿​‍‌‍‌﻿​​‌‍​‌‌﻿‌​‌‍‍​​﻿﻿‌‌‍​‍‌‍﻿﻿‌‍‌​‌﻿‍‌​‍‌‌​﻿‌‌‌​​‍‌‌﻿﻿‌‍‍﻿‌‍‌‌‌﻿‍‌​‍‌‌​﻿​﻿‌​‌​​‍‌‌​﻿​﻿‌​‌​​‍‌‌​﻿​‍​﻿​‍​﻿​‌​﻿‌‍​﻿​‌‌‍‌‌​﻿​‌​﻿‍‌‌‍​﻿‌‍‌‌​﻿​‌‌‍​﻿​﻿‌‍‌‍​‍​‍‌‌​﻿​‍​﻿​‍​‍‌‌​﻿‌‌‌​‌​​‍﻿‍‌‍​﻿‌‍‍​‌‍‍‌‌‍﻿​‌‍‌​‌﻿​‍‌‍‌‌‌‍﻿‍​‍‌‌​﻿‌‌‌​​‍‌‌﻿﻿‌‍‍﻿‌‍‌‌‌﻿‍‌​‍‌‌​﻿​﻿‌​‌​​‍‌‌​﻿​﻿‌​‌​​‍‌‌​﻿​‍​﻿​‍​﻿​﻿​﻿​‌‌‍​‌​﻿‍‌​﻿‍‌​﻿​‌​﻿​‍​﻿‍‌​﻿‌‌​﻿​‍​﻿​‌‌‍​‍​‍‌‌​﻿​‍​﻿​‍​‍‌‌​﻿‌‌‌​‌​​‍﻿‍‌﻿‌​‌‍‌‌‌﻿‍​‌﻿‌​​‍‌‍‌‍‍‌‌﻿​﻿‌​‌​‌﻿​‍‌‍​‌‌‍‌‍‌﻿‌​​﻿﻿‌​‍​‍‌﻿﻿‌</a><span style="color: rgb(0, 0, 0); font-family: Times; font-size: medium; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(248, 248, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial; display: inline !important; float: none;">, .​​​​‌﻿‍﻿​‍​‍‌‍﻿﻿‌﻿​‍‌‍‍‌‌‍‌﻿‌‍‍‌‌‍﻿‍​‍​‍​﻿‍‍​‍​‍‌﻿​﻿‌‍​‌‌‍﻿‍‌‍‍‌‌﻿‌​‌﻿‍‌​‍﻿‍‌‍‍‌‌‍﻿﻿​‍​‍​‍﻿​​‍​‍‌‍‍​‌﻿​‍‌‍‌‌‌‍‌‍​‍​‍​﻿‍‍​‍​‍​‍﻿﻿‌﻿​​‌﻿​‍‌‍‌‌‌﻿​﻿‌‍‌‌‌‍﻿‍‌﻿‌​‌‍​‌‌﻿‌​‌‍‍‌‌‍﻿﻿‌‍﻿‍​‍﻿﻿‌‍‍‌‌‍﻿‍‌﻿‌​‌‍‌‌‌‍﻿‍‌﻿‌​​‍﻿﻿‌‍‌‌‌‍‌​‌‍‍‌‌﻿‌​​‍﻿﻿‌‍﻿‌‌‍﻿﻿‌‍‌​‌‍‌‌​﻿﻿‌‌﻿​​‌﻿​‍‌‍‌‌‌﻿​﻿‌‍‌‌‌‍﻿‍‌﻿‌​‌‍​‌‌﻿‌​‌‍‍‌‌‍﻿﻿‌‍﻿‍​﻿‍﻿‌‍‍‌‌‍‌​​﻿﻿‌​﻿‌﻿​﻿​‌​﻿‌‍‌‍‌‌​﻿​‍‌‍​﻿​﻿‍​‌‍​‌​‍﻿‌‌‍​‌​﻿​‌​﻿‌‍​﻿‍​​‍﻿‌​﻿‌​​﻿‍​​﻿‌‌​﻿​‍​‍﻿‌​﻿‍​​﻿‌﻿‌‍‌‌​﻿​‍​‍﻿‌​﻿​​‌‍​﻿​﻿​‍‌‍​‍​﻿‌​‌‍‌​​﻿‌​​﻿​‍‌‍​﻿​﻿‍‌​﻿‌​‌‍​‍​﻿‍﻿‌﻿‌​‌﻿‍‌‌﻿​​‌‍‌‌​﻿﻿‌‌﻿​﻿‌‍‍‌‌‍﻿‌‌﻿​​‌‍﻿​‌‍‌‌‌​​‍‌‍﻿​‌‍﻿﻿‌‍​﻿‌‍‍﻿​﻿‍﻿‌﻿​​‌‍​‌‌﻿‌​‌‍‍​​﻿﻿‌‌‍​‍‌‍﻿﻿‌‍‌​‌﻿‍‌​‍‌‌​﻿‌‌‌​​‍‌‌﻿﻿‌‍‍﻿‌‍‌‌‌﻿‍‌​‍‌‌​﻿​﻿‌​‌​​‍‌‌​﻿​﻿‌​‌​​‍‌‌​﻿​‍​﻿​‍​﻿​‌​﻿‌‍​﻿​‌‌‍‌‌​﻿​‌​﻿‍‌‌‍​﻿‌‍‌‌​﻿​‌‌‍​﻿​﻿‌‍‌‍​‍​‍‌‌​﻿​‍​﻿​‍​‍‌‌​﻿‌‌‌​‌​​‍﻿‍‌‍​﻿‌‍‍​‌‍‍‌‌‍﻿​‌‍‌​‌﻿​‍‌‍‌‌‌‍﻿‍​‍‌‌​﻿‌‌‌​​‍‌‌﻿﻿‌‍‍﻿‌‍‌‌‌﻿‍‌​‍‌‌​﻿​﻿‌​‌​​‍‌‌​﻿​﻿‌​‌​​‍‌‌​﻿​‍​﻿​‍‌‍‌​​﻿​‍‌‍​‌​﻿‍​‌‍‌‍‌‍‌‌​﻿‌‍‌‍​‍​﻿​‌‌‍​‍‌‍‌‌​﻿‌﻿​‍‌‌​﻿​‍​﻿​‍​‍‌‌​﻿‌‌‌​‌​​‍﻿‍‌﻿‌​‌‍‌‌‌﻿‍​‌﻿‌​​﻿﻿﻿‌‍​‍‌‍​‌‌﻿​﻿‌‍‌‌‌‌‌‌‌﻿​‍‌‍﻿​​﻿﻿‌​‍‌‌​﻿​‍‌​‌‍‌﻿​​‌﻿​‍‌‍‌‌‌﻿​﻿‌‍‌‌‌‍﻿‍‌﻿‌​‌‍​‌‌﻿‌​‌‍‍‌‌‍﻿﻿‌‍﻿‍​‍‌‍‌‍‍‌‌‍‌​​﻿﻿‌​﻿‌﻿​﻿​‌​﻿‌‍‌‍‌‌​﻿​‍‌‍​﻿​﻿‍​‌‍​‌​‍﻿‌‌‍​‌​﻿​‌​﻿‌‍​﻿‍​​‍﻿‌​﻿‌​​﻿‍​​﻿‌‌​﻿​‍​‍﻿‌​﻿‍​​﻿‌﻿‌‍‌‌​﻿​‍​‍﻿‌​﻿​​‌‍​﻿​﻿​‍‌‍​‍​﻿‌​‌‍‌​​﻿‌​​﻿​‍‌‍​﻿​﻿‍‌​﻿‌​‌‍​‍​‍‌‍‌﻿‌​‌﻿‍‌‌﻿​​‌‍‌‌​﻿﻿‌‌﻿​﻿‌‍‍‌‌‍﻿‌‌﻿​​‌‍﻿​‌‍‌‌‌​​‍‌‍﻿​‌‍﻿﻿‌‍​﻿‌‍‍﻿​‍‌‍‌﻿​​‌‍​‌‌﻿‌​‌‍‍​​﻿﻿‌‌‍​‍‌‍﻿﻿‌‍‌​‌﻿‍‌​‍‌‌​﻿‌‌‌​​‍‌‌﻿﻿‌‍‍﻿‌‍‌‌‌﻿‍‌​‍‌‌​﻿​﻿‌​‌​​‍‌‌​﻿​﻿‌​‌​​‍‌‌​﻿​‍​﻿​‍​﻿​‌​﻿‌‍​﻿​‌‌‍‌‌​﻿​‌​﻿‍‌‌‍​﻿‌‍‌‌​﻿​‌‌‍​﻿​﻿‌‍‌‍​‍​‍‌‌​﻿​‍​﻿​‍​‍‌‌​﻿‌‌‌​‌​​‍﻿‍‌‍​﻿‌‍‍​‌‍‍‌‌‍﻿​‌‍‌​‌﻿​‍‌‍‌‌‌‍﻿‍​‍‌‌​﻿‌‌‌​​‍‌‌﻿﻿‌‍‍﻿‌‍‌‌‌﻿‍‌​‍‌‌​﻿​﻿‌​‌​​‍‌‌​﻿​﻿‌​‌​​‍‌‌​﻿​‍​﻿​‍‌‍‌​​﻿​‍‌‍​‌​﻿‍​‌‍‌‍‌‍‌‌​﻿‌‍‌‍​‍​﻿​‌‌‍​‍‌‍‌‌​﻿‌﻿​‍‌‌​﻿​‍​﻿​‍​‍‌‌​﻿‌‌‌​‌​​‍﻿‍‌﻿‌​‌‍‌‌‌﻿‍​‌﻿‌​​‍‌‍‌‍‍‌‌﻿​﻿‌​‌​‌﻿​‍‌‍​‌‌‍‌‍‌﻿‌​​﻿﻿‌​‍​‍‌﻿﻿‌</span>` // eslint-disable-line
```

Comment:
Noticed that somewhere between pushing and creating the PR it's automatically removing the unicodes from the input for the test which is an issue :(